TOC-tidb-cloud.md

```diff
@@ -181,7 +181,7 @@
     - [Password Authentication](/tidb-cloud/tidb-cloud-password-authentication.md)
     - [Identity Access Management](/tidb-cloud/manage-user-access.md)
   - Network Access Control
-    - [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md)
+    - [TLS Connect to Serverless Tier](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md)
```

Comment:
Please change the file name synchronously.

Reply:
Maybe change the file name will also change the link?

Reply:
Yes. If the meaning is not that different, we could still use the current filename.